src/main/java/uk/gov/hmcts/juror/api/moj/xerox/letters/ConfirmLetter.java

```diff
@@ -36,14 +36,13 @@ protected void setupWelsh() {
         addData(LetterDataType.DATE_OF_LETTER, 18);
         addData(LetterDataType.COURT_LOCATION_CODE, 3);
         addData(LetterDataType.WELSH_COURT_NAME, 40);
-        addData(LetterDataType.COURT_NAME, 40);
-        addData(LetterDataType.BUREAU_NAME, 35);
-        sharedBureauSetup();
-        addData(LetterDataType.BUREAU_POSTCODE, 10);
+        addData(LetterDataType.BUREAU_NAME, 40);
```

Comment:
is 40 correct? was previously 35 ... spec says 35

Reply:
This aligns to the heritage application we have a task post go live to review the spec misalignments